extensions/nextdns/src/components/list-item.tsx

```diff
@@ -0,0 +1,121 @@
+import { Action, ActionPanel, Alert, Color, confirmAlert, Icon, Keyboard, List, showToast, Toast } from "@raycast/api";
+import { removeDomain, toggleDomain } from "../libs/api";
+import { DomainListItem, Mutate } from "../types";
+import AddDomain from "./add-domain";
+import { getIconById } from "../libs/utils";
+
+export function ListItem(props: { domainItem: DomainListItem; mutate: Mutate; data: DomainListItem[] }) {
+  const { domainItem, mutate, data } = props;
+
+  return (
+    <List.Item
+      title={`*.${domainItem.id}`}
+      icon={getIconById(domainItem.id)}
+      accessories={[
+        {
+          tag: domainItem.active
+            ? { color: Color.Green, value: "Activated" }
+            : { color: Color.Red, value: "Deactivated" },
+        },
+      ]}
+      actions={<Actions item={domainItem} mutate={mutate} data={data} />}
+    />
+  );
+}
+
+function Actions({ item, mutate, data }: { item: DomainListItem; mutate: Mutate; data: DomainListItem[] }) {
+  async function toggle(item: DomainListItem) {
+    const newStatus = !item.active;
+    const { id, type } = item;
+
+    const toast = await showToast({
+      style: Toast.Style.Animated,
+      title: newStatus ? "Activating domain" : "Deactivating domain",
+      message: id,
+    });
+    try {
+      await mutate(toggleDomain({ element: { id, type, active: newStatus } }), {
+        optimisticUpdate(data) {
+          const { result, profileName } = data;
+          const index = result.findIndex((item) => item.id === id);
+          result[index].active = newStatus;
+          return { result, profileName };
+        },
+      });
+      toast.style = Toast.Style.Success;
+      toast.title = newStatus ? "Activated domain" : "Deactivated domain";
+    } catch (error) {
+      toast.style = Toast.Style.Failure;
+      toast.title = `Could not ${newStatus ? "activate" : "deactivate"} domain`;
+    }
+  }
+
+  async function removeItem(element: DomainListItem) {
+    await confirmAlert({
+      title: "Are you sure?",
+      icon: Icon.Trash,
+      message: `This will remove it from the ${element.type}list`,
```

Comment:
```suggestion
      message: `This will remove it from the ${element.type} list`,
```

Reply:
It's actually the name they're using 🙂

<img width="318" alt="image" src="https://github.com/user-attachments/assets/83722d6b-12b5-45f8-be71-4430cf87b93b">